src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * Deprecate undefined and numeric keys with `service_locator` config
  * Fail if Target attribute does not exist during compilation
  * Enable deprecating parameters with `ContainerBuilder::deprecateParameter()`
+ * Allow to trim XML service parameters value by using `trim-value="true"` attribute
```

Comment:
why not just trim instead of trim-value BTW?

Reply:
Thought that `trim` maybe wasn't explicit enough. But I definitely prefer `trim`, it's updated as well